tests/pylint/test_enforce_type_hints.py

```diff
@@ -1228,6 +1219,15 @@ async def test_sample( #@
             end_line=6,
             end_col_offset=36,
         ),
+        pylint.testutils.MessageTest(
```

Comment:
The order of the messages has changed, as `hass` now gets check in the same order as `_TEST_FIXTURES`, and not as a preliminary check of `_COMMON_ARGUMENTS`